src/Annotation/VoteUpDownWidget.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\vud\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Vote Up/Down Widget annotation object.
+ *
+ * @see \Drupal\vud\Plugin\VoteUpDownFieldManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class VoteUpDownWidget extends Plugin {
+
+  /**
+   * The plugin ID.
+   *
+   * @var string
+   */
+  public $plugin_id;
+
+  /**
+   * The label of the plugin.
+   *
+   * @var \Drupal\Core\Annotation\Translation
+   *
+   * @ingroup plugin_translatable
+   */
+  public $label;
+
+  /**
+   * Filename of the plugin used.
+   *
+   * @var string
+   */
+  public $widget_template;
```

Comment:
Annotation properties may have default values.

In D7, all widgets have the widget template named `widget.tpl.php`.

Following that, now I think all widgets can have `widget.html.twig`, which means we can set the default value for the widget_template property to `widget`, instead of defining it on every widget plugin declaration.

This of course means that the template files need to be renamed.